docs/style_guide.md

```diff
@@ -159,7 +159,7 @@ The goal of coding standards is to make code easier to understand and maintain.
    - `--cheese`
 
 
-# Other ways to make your code better
+## Other ways to make your code better
```

Comment:
I cannot believe that I didn't know that "The keywords `and`, `or`, and `not` can be used instead of `&&`, `||`, and `!` in C++." I feel like I have never seen the keywords used in example code ever. Very cool.